src/t8_forest/t8_forest_adapt.cxx

```diff
@@ -406,15 +419,16 @@ t8_forest_adapt (t8_forest_t forest)
   t8_tree_t           tree;
   t8_tree_t           tree_from;
   sc_list_t          *refine_list = NULL;       /* This is only needed when we adapt recursively */
-  size_t              num_children; 
-  size_t              num_siblings; 
-  size_t              zz; 
+  size_t              num_children;
```

Comment:
The types of these variables and the return-type of the element_functions do not match. Please use the same types.

Reply:
Should be fine now.